### PR TITLE
chore: add source to transcript card

### LINKED ISSUE #58: Update Transcript Cards in "Dive into Transcripts" Section
We recently updated the design for the "Dive into Transcripts" section in the Bitcoin Transcript Registry. In this update, each transcript card now includes the **source** of the transcript, in addition to the **title** and **speakers**. This change makes it easier for users to identify where each transcript originates.

Please see the [designs](https://www.figma.com/design/KUfTWoQWvLjuBbDf0EmvIc/BTC-Transcripts-Main-File?node-id=1271-2055&t=wR908My7pxOCNHNs-1) for the updated look and layout.

dive-into-transcripts-section

### PR DESCRIPTION
**Pull request by IgboPharaoh**
fixes #58

### CODE REVIEW

File: src/components/landing-page/TranscriptCard.tsx [hunk 1]
```diff
@@ -13,16 +14,23 @@ interface TranscriptCardProps {
 const TranscriptCard = ({ data, daysOpened, transcripts }: TranscriptCardProps) => {
   const remainingSpeakers = data?.speakers?.length && data?.speakers.length > 2 ? data?.speakers.length - 2 : 0;
 
+  const mappedKeyToName = SourceCountData.find((source) => source.slug === data.slugAsParams[0])?.name;
```
Comment (Emmanuel-Develops) [inline src/components/landing-page/TranscriptCard.tsx:17]: can we move this to the page component and pass the data as props

**Review by Emmanuel-Develops — CHANGES_REQUESTED** [review]

File: src/components/landing-page/featured-transcripts/FeaturedTranscriptClient.tsx [hunk 2]
```diff
@@ -18,6 +19,9 @@ const FeaturedTranscriptClient = ({
     setFeatured([...featuredTranscripts].sort(() => 0.5 - Math.random()));
   };
 
+  const getSourceFromTranscript = (data: Transcript) =>
```
Comment (Emmanuel-Develops) [inline src/components/landing-page/featured-transcripts/FeaturedTranscriptClient.tsx:22]: I meant, we get this data on the server component before it gets to this client component.

We should always offload logic to server components when we can.
For best performance we would have to modify the functions that get "featured" and "latest" transcripts.

home page load doesn't indicate any extra processing, further optimizations should in a separate PR for complete evaluation

**Review by kouloumos — COMMENTED** [review]
@IgboPharaoh , @Emmanuel-Develops , the cards in the "Dive into Transcripts" section are not clickable anymore and I think that this is the PR that introduced the issue into production.
Clicking on the cards doesn't work in the preview of this PR.
Needs further investigation.

**Comment by IgboPharaoh** [comment]
Great catch andreas, the issue is definitely from this pr and the mistake is from my end. I'll fix it shortly. 

Thanks

**Comment by IgboPharaoh** [comment]
> @IgboPharaoh , @Emmanuel-Develops , the cards in the "Dive into Transcripts" section are not clickable anymore and I think that this is the PR that introduced the issue into production. Clicking on the cards doesn't work in the preview of this PR. Needs further investigation.

#68 fixes this issue

**Pull request merged**